CMakeLists.txt

```diff
@@ -15,9 +15,6 @@ set(CMAKE_C_COMPILER_WORKS 1)
 set(CMAKE_EXPORT_COMPILE_COMMANDS TRUE)
 set(CMAKE_VERBOSE_MAKEFILE TRUE)
 
-# SCHEREMO: Help most IDE's LSPs find definitions
-set(CMAKE_EXPORT_COMPILE_COMMANDS 1)
-
```

Comment:
Why remove this?

Reply:
Because it is duplicated. The other definition is just three lines above